homeassistant/components/media_player/harman_kardon_avr.py

```diff
@@ -0,0 +1,132 @@
+"""
+Support for interface with an Harman/Kardon or JBL AVR.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.harman_kardon_avr/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.media_player import (
+    SUPPORT_TURN_OFF, SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_STEP,
+    PLATFORM_SCHEMA, SUPPORT_TURN_ON, SUPPORT_SELECT_SOURCE,
+    MediaPlayerDevice)
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME, CONF_PORT, STATE_OFF, STATE_ON)
+
+REQUIREMENTS = ['hkavr==0.0.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Harman Kardon AVR'
+DEFAULT_PORT = 10025
+
+SUPPORT_HARMAN_KARDON_AVR = SUPPORT_VOLUME_STEP | SUPPORT_VOLUME_MUTE | \
+                            SUPPORT_TURN_OFF | SUPPORT_TURN_ON | \
+                            SUPPORT_SELECT_SOURCE
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+})
+
+
+def setup_platform(hass, config, add_devices, discover_info=None):
```

Comment:
Please rename `add_devices` to `add_entities`.
```suggestion
def setup_platform(hass, config, add_entities, discover_info=None):
```